cartoframes/dataset.py

```diff
@@ -153,6 +152,9 @@ def exists(self):
             self.cc._debug_print(err=err)
             return False
 
+    def get_query(self):
```

Comment:
This is required to solve the case of visualizing a Dataset.from_table(). The Dataset should contain the logic because it owns the context, which is needed to get the correct schema. This method is also useful to refactor the `compute_geom_type` method.

We can move it to utils `get_query(dataset)` but IMO it makes sense to keep it in the Dataset class, maybe with a better name.

Reply:
Looking at the whole PR, this method helps to clean the `source` module, so 👍 

As @oleurud suggested, instead of being part of the Dataset API it could be a helper function inside the cartoframes.datasets namespace, something like `get_query(dataset)`. At least we'd keep the public Dataset API a little bit cleaner.

My 2 cents.

Reply:
Yes, I see now that this is part of the public API, so the `get_query(dataset)` method makes more sense ;)